index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang='en'>
+    <head>
+        <meta charset="UTF-8">
+        <title>WEATHER REPORT</title>
+        <link href="./styles/index.css" rel='stylesheet'>
+        <script src="https://kit.fontawesome.com/73e38c15ac.js" crossorigin="anonymous"></script>
+
+    </head>
+    <body>
+        <div id="landscape" class="sunny">
+            <section id="cityName">
+                <span >
+                    <h2 id="headCityName" class='stormy'>SEATTLE </h2>
+                </span>
```

Comment:
This was an extraneous `<span>` element, also, since there is an `id` on the section you can select elements within it using that (see stylesheet):

```suggestion
                <h2>SEATTLE</h2>
```

Also, because the enclosing `#landscape` has the weather class attached to it you don't need that here.